SPECS/perl/perl.spec

```diff
@@ -177,6 +177,7 @@ Patch201:       perl-5.16.3-Link-XS-modules-to-libperl.so-with-EU-MM-on-Linux.pa
 Patch202:       CVE-2023-47100.patch
 Patch203:       CVE-2023-31486.patch
 Patch204:       CVE-2023-31484.patch
+Patch205:       CVE-2023-47038.patch
```

Comment:
Looks like this package hasn't been moved to `autosetup` or `autopatch` yet, can you either add the explicit patch directive, or switch it over if it looks doable?